controllers/gce/firewalls/fakes.go

```diff
@@ -18,17 +18,16 @@ package firewalls
 
 import (
 	"fmt"
-	"strconv"
 
 	compute "google.golang.org/api/compute/v1"
-	netset "k8s.io/kubernetes/pkg/util/net/sets"
 
 	"k8s.io/ingress/controllers/gce/utils"
 )
 
 type fakeFirewallsProvider struct {
-	fw    map[string]*compute.Firewall
-	namer *utils.Namer
+	fw         map[string]*compute.Firewall
+	namer      *utils.Namer
+	networkUrl string
```

Comment:
I think `namer` has to be removed from the fakeFirewallsProvider. Cloudprovider should no longer be generating the name given a suffix - it should just be given the name.